### DISCUSSION TITLE
Change configuration for not change token

### DISCUSSION THREAD

**Discussion by Leprechault** [community/community#22411] [New to GitHub]
<p>I have a private repository and access of the <code>raw.githubusercontent.com</code> by API is using the <code>?token=?????</code> in the end of the file name. But this <code>token</code> change during the time (10 -15 days) and this is not so good for my purposes. I don’t find any way to do not change the <code>token</code> information. Please, this kind of configuration is possible?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You need to create a <a href="https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token">PAT</a> or OAuth token (technically the PAT is an OAuth token, but one you can get without implementing an OAuth client) and send it in the <code>Authorization</code> header instead of in the URL, as <a href="https://docs.github.com/en/rest/overview/resources-in-the-rest-api#authentication">described in the “Authentication” section of the API documentation</a>. That will work as long as the token is valid.</p>
